pokemon-minisite/src/app/app.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+
+import { PokemonListComponent } from './components/pokemon-list/pokemon-list.component';
+import { HeaderComponent } from './components/header/header.component';
+
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet, HeaderComponent, PokemonListComponent],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent implements OnInit{
+  title = 'pokemon-minisite';
+
+  ngOnInit(): void {
```

Comment:
delete oninit function and import if not in use

Reply:
done